oioioi/forum/utils.py

```diff
@@ -68,6 +69,9 @@ def is_proper_forum(request, *args, **kwargs):
 def can_interact_with_users(request):
     if request.user.is_anonymous:
         return False
+    if request.contest.forum.only_for_registered and \
+            (not is_participant(request) and not is_contest_admin(request)):
+        return False
```

Comment:
Imo `is_contest_basicadmin` or `can_admin_contest` would be better here.

Reply:
Many functions in forum app already use `is_contest_admin`, even in this function
```py
    return is_contest_admin(request) or (not is_banned and not is_locked)
```

Reply:
I am for consistency. In case of doubts I would suggest to move it to an issue.